spec.html

```diff
@@ -37841,7 +37683,7 @@ <h1>Abstract Operations for TypedArray Objects</h1>
       <emu-clause id="typedarray-species-create" type="abstract operation">
         <h1>
           TypedArraySpeciesCreate (
-            _exemplar_: unknown,
+            _exemplar_: an Object that has [[TypedArrayName]] and [[ContentType]] internal slots,
```

Comment:
```suggestion
            _exemplar_: a TypedArray,
```

Though, we actually have a definition for `Integer-Indexed exotic object`, unlike `TypedArray`, so possibly we should be using the former everywhere?